avaframe/runProbAnalysisOnly.py

```diff
@@ -17,14 +19,16 @@
 from avaframe.out3Plot import statsPlots as sP
 
 
-def runProbAna(avalancheDir=""):
+def runProbAna(avalancheDir="", modName="com1DFA"):
     """Run a com1DFA probability analysis with parameters and only an
     avalanche directory as input
 
     Parameters
     ----------
     avalancheDir: str
         path to avalanche directory (setup e.g. with init scripts)
+    modName: str
+        name of computational module that has been used to produce sim results
```

Comment:
Same here, quick comment about why this is here?

Reply:
added info